shared/js/initApp.js

```diff
@@ -286,8 +287,18 @@ if (appOptions.droplet) {
   promise = loadSource('blockly')()
     .then(loadSource(appOptions.locale + '/blockly_locale'));
   if (appOptions.level.isProjectLevel) {
+    // example paths:
+    // edit: /p/artist#7uscayNy-OEfVERwJg0xqQ==/edit
+    // view: /p/artist#7uscayNy-OEfVERwJg0xqQ==
     var app_id = location.hash.slice(1);
     if (app_id) {
+      // TODO ugh, we should use a router. maybe we should use angular :p
+      var params = app_id.split("/")
+      if (params.length > 1 && params[1] == "edit") {
```

Comment:
Could do something like `/\/edit$/.test(location.hash)` but what you have is fine.  Also I don't think we want to call it `app_id` yet on line 294.  Maybe a temp var until we can split out the real `app_id`?